samples/testing-frameworks/appium/client-side/java/src/test/resources/desiredCapabilities.androidBiometry.clientside.properties

```diff
@@ -1,9 +1,10 @@
 platformName=Android
-deviceName=Android Device
 fullReset=false
```

Comment:
`appium_fullReset`

Reply:
adds this in the getDesiredCapabilitiesFromProperties()

Reply:
Hm ok can you link where does it happen in code please?

Reply:
never mind, my mistake, corrected